src/modules/auth/credentials/BulkAssignmentContainer.js

```diff
@@ -0,0 +1,453 @@
+import React, { useState, useEffect, useMemo } from "react"
+import { useTranslation } from 'react-i18next';
+import { Link } from "react-router-dom";
+import { Card, CardBody, CardHeader,
+	CardFooter, Button, ButtonGroup } from "reactstrap";
+import {DataTable, ButtonWithAuthz } from 'asab-webui';
+import RoleDropdown from "../components/RoleDropdown";
+import { useSelector } from "react-redux";
+
+const BulkAssignmentContainer = (props) => {
+
+	const { t } = useTranslation();
+
+	const [data, setData] = useState([]);
+	const [count, setCount] = useState(0);
+	const [page, setPage] = useState(1);
+	const [limit, setLimit] = useState(0);
+	const [loading, setLoading] = useState(true);
+	const [credentialsFilter, setCredentialsFilter] = useState("");
+	const [selectedCredentials, setSelectedCredentials] = useState([]);
+
+	const [tenants, setTenants] = useState({});
+	const [tenantsPage, setTenantsPage] = useState(1);
+	const [tenantsCount, setTenantsCount] = useState(0);
+	const [tenantsLimit, setTenantsLimit] = useState(0);
+	const [tenantsFilter, setTenantsFilter] = useState("");
+	const [loadingTenants, setLoadingTenants] = useState(true);
+	const [selectedTenants, setSelectedTenants] = useState([{_id: 'Global roles', selectedRole: []}]);
+
+	const [show, setShow] = useState(false);
+	const [showTenantsContentLoader, setShowTenantsContentLoader] = useState(false);
+
+	const resource = "authz:superuser";
+	const resources = useSelector(state => state.auth?.resources);
+
+	// headers for Credentails List
+	const headers = [
+		{
+			name: '',
+			customComponent: {
+				generate: (credential) => (
+					<div>
+						<ButtonWithAuthz
+							title={t(`BulkAssignmentContainer|${credential.assigned ? "Credential already assigned" : "Add credential"}`)}
+							id={credential._id}
+							size="sm"
+							color="primary"
+							outline
+							onClick={() => saveToSelectedCredentials(credential)}
+							resource={resource}
+							resources={resources}
+							disabled={credential.assigned}
+						>
+							<i className="cil-plus"></i>
+						</ButtonWithAuthz>
+					</div>
+				)
+			}
+		},
+		{
+			name: '',
+			customComponent: {
+				generate: (obj) => (
+					<div className="no-wrap-40ch">
+						{obj.suspended === true ?
+							<span className="cil-user-unfollow text-muted mr-1" title={(obj.registered === false) ? t("BulkAssignmentContainer|Credentials invited") : t("BulkAssignmentContainer|Credentials suspended")}/>
+							: <span className="cil-user mr-1" />}
+						<Link
+							style={{color: obj.suspended === true && '#73818f'}}
```

Comment:
@petrKavulok I think this color is `secondary`, or not? I think we should avoid use of hex colors directly in the code.

Reply:
Take a look how this is handled in credentials list container